packages/react/src/patterns/blocks/ContentBlockSimple/README.md

```diff
@@ -0,0 +1,84 @@
+# Content Block - Simple
+
+> The Content Block - Simple pattern is a decorator of `ContentBlock`, and
+> includes a single `ContentItem`, optional media (image), and ends with a CTA.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/content-block-simple';
+```
+
+> 💡 Only import fonts once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { SimpleLongForm } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <ContentBlockSimple
+      heading={heading}
+      copy={copy}
+      items={items}
+      mediaData={mediaData}
+      mediaType={mediaType}
+      cta={cta}
+    />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the ContentBlockSimple styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                                              |
+| --------- | -------- | --------- | ------------- | ------------------------------------------------------------------------ |
+| `heading` | YES      | String    | n/a           | Title of the content block.                                              |
+| `copy`    | YES      | String    | n/a           | Simple content item.                                                     |
+| `image`   | NO       | Array     | n/a           | Optional image to use after content item. See `image` section.           |
```

Comment:
Instead of an image section, maybe we can link to the Image documentation.

Reply:
I totally agree with this. The only reason I'm reluctant to do it now is because pretty much all the other READMEs are broken up into sections. I'll do it now but we should keep track of this and change them all in the future.

Reply:
This is part of the ontology work. Every README moving forward we've been trying to making this happen.